src/helpers/github.ts

```diff
@@ -99,3 +99,66 @@ export async function isCiGreen({ octokit, logger, env }: Context, sha: string,
     return false;
   }
 }
+
+function parseTarget({ payload }: Context, target: string) {
+  const owner = payload.repository.owner || "";
+  const [orgParsed, repoParsed] = target.split("/");
+  let repoTarget = null;
+  if (orgParsed !== owner) {
+    return null;
+  } else if (repoParsed) {
+    repoTarget = repoParsed;
+  }
+  return { org: owner, repo: repoTarget };
+}
+
+/**
+ * Returns all the pull requests that are opened and not a draft from the list of repos / organizations.
+ *
+ * https://docs.github.com/en/search-github/searching-on-github/searching-issues-and-pull-requests#search-only-issues-or-pull-requests
+ */
+export async function getOpenPullRequests(context: Context, targets: ReposWatchSettings) {
+  const { octokit, logger } = context;
+  const filter = [
+    ...targets.monitor.reduce<string[]>((acc, curr) => {
+      const parsedTarget = parseTarget(context, curr);
+      if (parsedTarget) {
+        return [...acc, parsedTarget.repo ? `repo:${parsedTarget.org}/${parsedTarget.repo}` : `org:${parsedTarget.org}`];
+      }
+      return acc;
+    }, []),
+    ...targets.ignore.reduce<string[]>((acc, curr) => {
+      const parsedTarget = parseTarget(context, curr);
+      if (parsedTarget) {
+        return [...acc, parsedTarget.repo ? `-repo:${parsedTarget.org}/${parsedTarget.repo}` : `-org:${parsedTarget.org}`];
+      }
+      return acc;
+    }, []),
+  ];
+  try {
+    const { data } = await octokit.rest.search.issuesAndPullRequests({
```

Comment:
sorry maybe I wasn't clear, you still need to use paginate, just that `GET /search/issue` can be replaced with `octokit.rest.search.issuesAndPullRequests`

Reply:
Oh I thought it would be automatically paginated after adding the plugin, will change then.